apollo/schema.sql

```diff
@@ -623,7 +623,8 @@ CREATE TABLE public.supported_products_rh_mirrors (
     match_variant text NOT NULL,
     match_major_version numeric NOT NULL,
     match_minor_version numeric,
-    match_arch text NOT NULL
+    match_arch text NOT NULL,
+    active boolean DEFAULT true NOT NULL
```

Comment:
I don't think this backfills in all databases. You may want to use your migration to add data to the column.